File: cpp/programmers/두 정수 사이의 합.cpp
#include <string>
#include <vector>

using namespace std;

long long solution(int a, int b) {
    long long answer = 0;
    if (a == b){
        answer = a;
    }
    else{
        if (a > b){
            int num_tmp = a;
            a = b;
            b = num_tmp;
        }
        for (int i=a; i<b+1; ++i){
            answer += i;
        }
    }
    return answer;
}

File: python/programmers/두 정수 사이의 합.py
# 프로그래머스 - 두 정수 사이의 합

# v1
# def solution(a, b):
#     res = 0
#     if a == b:
#         res = a
#     else:
#         if a > b:
#             tmp = a
#             a = b
#             b = tmp
#         for i in range(a,b+1):
#             res += i
#     return res

# v2
def solution(a, b):
    res = 0
    if a == b:
        res = a
    else:
        if a > b:
            a,b = b, a
        for i in range(a,b+1):
            res += i
    return res